mod-source-record-manager-server/src/main/java/org/folio/services/mappers/processor/functions/NormalizationFunction.java

```diff
@@ -351,6 +353,52 @@ public String apply(RuleExecutionContext context) {
         .map(AlternativeTitleType::getId)
         .orElse(STUB_FIELD_TYPE_ID);
     }
+  },
+
+  SET_ISSUANCE_MODE_ID() {
+
+    @Override
+    public String apply(RuleExecutionContext context) {
+      String subFieldValue = context.getSubFieldValue();
+      char seventhChar = subFieldValue.charAt(6);
+      List<IssuanceMode> issuanceModes = context.getMappingParameters().getIssuanceModes();
+      if (issuanceModes == null || issuanceModes.isEmpty()) {
+        return StringUtils.EMPTY;
```

Comment:
log debug cases with StringUtils.EMPTY

Reply:
Yes, sure. I added debug logging for each case where we will have an empty string as a result of normalization function.